helios-ts/lib.ts

```diff
@@ -150,6 +157,10 @@ export class HeliosProvider {
         const block = await this.#client.get_block_by_number(req.params[0], req.params[1]);
```

Comment:
Can we fix this? Looks like its accidentally returning blocks.

Reply:
<img width="1176" alt="image" src="https://github.com/user-attachments/assets/f0c5f1b5-2ba3-4a8b-959b-b28043ea796a" />

`eth_getBlockByNumber` seems to be fine.

Reply:
Oh my bad the diff made it look like `eth_uninstallFilter` was returning blocks.